data/json/items/gun/shot.json

```
@@ -766,4 +766,32 @@
     "flags": [ "RELOAD_ONE" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "shot": 6 } } ]
   }
+  {
+    "id": "makeshift_shotgun",
+    "copy-from": "shotgun_base",
+    "type": "GUN",
+    "name": { "str": "makeshift shotgun" },
+    "description": "A crude shotgun, composed of two thick steel pipes, an end cap and a nail. The lack of sights make this weapon only useful at point-blank range.",
+    "weight": "3629 g",
+    "volume": "2264 ml",
+    "looks_like": "remington_870",
+    "price": 8500,
+    "price_postapoc": 1000,
+    "bashing": 8,
+    "material": [ "steel"],
+    "ranged_damage": { "damage_type": "bullet", "amount": -2 },
+    "dispersion": 960,
+    "durability": 3,
+    "clip_size": 1,
+    "modes": [ [ "DEFAULT", "single", 1 ] ],
+    "valid_mod_locations": [
+      [ "barrel", 1 ],
+      [ "sling", 1 ],
+      [ "brass catcher", 1 ],
+      [ "sights mount", 1 ],
+      [ "underbarrel", 1 ]
+    ],
+    "flags": [ "RELOAD_ONE" ],
```

```suggestion
      [ "sights mount", 1 ],
      [ "underbarrel mount", 1 ]
    ],
    "flags": [ "RELOAD_EJECT" ],
```
1. Brass catcher only applies to guns with extractors. Slamfire shotgun gets RELOAD_EJECT.
2. No native Picatinny or NATO accessory rail on the bottom = it needs a mount slot.